### PR TITLE
android-privacy: Labelling in context

### PR DESCRIPTION
**Pull request by imsodin**
The saga goes on. The supports latest reply:

> Hi developers at Syncthing Community,
> 
> Thanks again for contacting the Google Play team.
> 
> Unfortunately, I’m not able to provide any more detail or a better answer to your question.
> 
> As I mentioned in our previous email, the "Privacy Policy" is **clearly labeled in not only the title but context of content**, even translated in English.
> 
> Please fix this issue and if you think your app is in compliance, **please submit your app for another review**. You may want to review the [Developer Program Policies](https://play.google.com/about/developer-content-policy.html) for additional policy guidance.
> 
> If you have a different question about the Google Play Policy, please let me know.
> 
> Regards,
> Scott
> The Google Play Team

My previous email:

> Hi Scott,
> 
> I'll reply in line with quotes from our privacy policy:
> 
> On 03/03/2021 02:26, [googleplay-developer-support@google.com](mailto:googleplay-developer-support@google.com) wrote:
> 
> > **Invalid privacy policy**
> >
> >
> >
> >
> >
> > Based on our review, your privacy policy link is updated **in the designated field** in the Play Console but doesn’t comply with our policy requirements. Please [add or update your privacy policy](https://support.google.com/googleplay/android-developer/answer/113469#privacy), and make sure it is available on an active URL (no PDFs), is non-editable, applies to your app, and **specifically** covers user privacy including your app’s usage of location data.
> 
> The very first paragraph of our privacy policy specifically covers the user privacy with regarde to our usage of location data:
> 
> > The app does not store any location data. The permission to access location in the background is used only to check for an SSID of the currently active Wi-Fi. The sole purpose of this information is to limit synchronisation to user-configured Wi-Fi connections. The app itself never collects the location in the process, and as such it also does not use or save any location data.
> Please explain how this is not specific enough.
> 
> > **Please make sure** privacy policy is clearly labeled as a privacy policy in the context of the content and specifies handling of location data.
> Our privacy policy is named "Syncthing-Android Privacy Policy" and talks about "the app Syncthing-Android" and "location data". That seems like a pretty good labelling, as it is the same labelling Google Play itself uses. Would a sub-header like "Handling of location data" help?
> 
> Best,
> Simon

I am pretty much at my wits end. This sentence doesn't make much sense to me. Definitely not the last part about translation (where does translation come into this?), but also the rest seems meaningless (and ironically bad English) to me.

> "Privacy Policy" is **clearly labeled in not only the title but context of content**, even translated in English.

I sprinkled the term "user privacy" in the text for context, maybe that's what they want?

@tomasz1986 What do you think?

### CODE REVIEW

File: content/android-privacy-policy.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Syncthing-Android Privacy Policy
 
-The app syncthing-android does not collect any user information.
+The app syncthing-android values user privacy and thus does not collect any user information.
 
-The app does not store any location data.
+As location data is very sensitive personal information relating to your privacy, the app does not store any location data.
```
Comment (tomasz1986) [inline content/android-privacy-policy.md]: For location data includes very sensitive personal information related to user privacy, the app does not store any location data.

File: content/android-privacy-policy.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 It requires the permission to access the location in the background, to look up the currently active Wi-Fi's SSID.
 That information is used to enable syncing only on user-configurable Wi-Fi's.
-The app never actually looks up your location in the process, and thus doesn't use or store location data.
+At no point is this information used to look up your actual location nor is t ever passed on to 3rd parties.
```
Comment (tomasz1986) [inline content/android-privacy-policy.md:6]: Just a typo here, "nor is t ever" -> "nor is it ever".

**Comment by tomasz1986** [comment]
I have just reworded one sentence a little bit to make it more formal. Other than that, I'm not sure. They are not saying what they actually want, and the e-mail is super vague, so I would say that we are in the dark here.

If this gets rejected too, then I would say that maybe the policy simply needs to be more detailed and longer, and also with some subheadings for each section (e.g. location, camera, etc.).

Another question is whether it is actually humans that review those policies, or maybe only bots that look for specific keywords and such 😉. If the required keywords are missing, then the policy gets rejected.

**Comment by imsodin** [comment]
I incorporated your changes - thanks. I didn't base it on current main, so had to rebase it which required a few more changes. Could you please look over it for any obvious mistakes again before I merge. 

At least the emails are definitely from a human - I missed one form (there's not just one) and didn't realise at first, and they got a bit frustrated over it (instead of just telling me: "You missed a form", which they only after a few back and forths).

File: content/android-privacy-policy.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
-Syncthing-Android does not collect any user information.
+Syncthing-Android values user privacy and thus does not collect any user information.
 
-The app does not store any location data.
+For location data includes very sensitive personal information related to user privacy, the app does not store any location data.
 The permission to access location in the background is used only to check for an SSID of the currently active Wi-Fi.
 The sole purpose of this information is to limit synchronisation to user-configured Wi-Fi connections.
-The app itself never collects the location in the process, and as such it also does not use or save any location data.
+At no point is this information used to look up your actual location nor is it ever passed on to 3rd parties.
```
Comment (tomasz1986) [inline content/android-privacy-policy.md]: At no point is this information used to track the user's actual location nor is it ever shared with any 3rd party entities.

**Comment by tomasz1986** [comment]
I have just made the last sentence a bit more formal. The rest sounds fine to me 👍.

> At least the emails are definitely from a human - I missed one form (there's not just one) and didn't realise at first, and they got a bit frustrated over it (instead of just telling me: "You missed a form", which they only after a few back and forths).

That is the good news then. At least they do have some actual people dealing with the users. I was just wondering about the review process itself, whether it was fully automatic, or if at any stage there was a human being involved.

File: content/android-privacy-policy.md (hunk 3 above)
Comment (imsodin) [reply content/android-privacy-policy.md]: What's the problem with "look up"? "Track" implies something more to me, something continuous. What it should say is that we don't look at their location, not even once. I changed it to "determine", that formal enough?

Comment (tomasz1986) [reply content/android-privacy-policy.md]: I was looking for a more formal word, but nothing else came to my mind. "Determine" is much better!

**Pull request merged**